### PR TITLE
SharedUserCollections: Views

### PR DESCRIPTION
**Pull request by eronisko**
Please uncomment the share link in `resources/views/frontend/user-collection/show.blade.php` to test this out.

### CODE REVIEW

File: resources/views/frontend/user-collection/show.blade.php [hunk 1]
```diff
@@ -19,9 +19,10 @@
                 <p>{!! trans('user-collection.content-usage') !!}</p>
                 <p>{!! trans('user-collection.content-feedback') !!}</p>
 
-                @unless($items->isEmpty())
-                <a href="{{ route('frontend.shared-user-collections.create', ['ids' => request()->ids ] ) }}">Zdieľať {{-- TODO i18n --}} </a>
-                @endunless
+                {{-- TODO enable on WEBUMENIA-1654 (shared user collections) is launched --}}
+                {{-- @unless($items->isEmpty())
+                <a href="{{ route('frontend.shared-user-collections.create', ['ids' => request()->ids ] ) }}">Zdieľať</a>
+                @endunless --}}
```
Comment (igor-kamil) [inline resources/views/frontend/user-collection/show.blade.php:25]: for testing you can enable it for auth users e.g.

```suggestion
                {{-- TODO enable on WEBUMENIA-1654 (shared user collections) is launched --}}
                @unless($items->isEmpty() && !Auth::check())
                <a href="{{ route('frontend.shared-user-collections.create', ['ids' => request()->ids ] ) }}">Zdieľať</a>
                @endunless
```

File: resources/js/components/user-collections/NavLink.vue [hunk 2]
```diff
@@ -15,23 +15,11 @@
...
-        data() {
-            return {
-                store
-            }
-        },
-        computed: {
-            href() {
-                const url = new URL(this.baseHref)
-                const ids = this.store.getItems().map(({id}) => id)
+        mixins: [require('./link-mixin')],
```
Comment (igor-kamil) [inline resources/js/components/user-collections/NavLink.vue:19]: 👌

File: resources/views/components/user-collections/share-form.blade.php [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+                            Tento výber budeš môcť naďalej <strong>upravovať</strong> na tejto adrese:
+                        </p>
+                        <copy-to-clipboard
+                            class="no-border"
+                            value="{{ route('frontend.shared-user-collections.edit', ['collection' => $collection, 'token' => $collection->update_token]) }}"
+                            button-label="{{ trans('general.copy') }}"
+                            success-text="{{ trans('general.copied_to_clipboard') }}"
+                        ></copy-to-clipboard>
+                        <p class="mt-2">
+                            (odlož si ju)
```
Comment (igor-kamil) [inline resources/views/components/user-collections/share-form.blade.php:136]: make this highlighted pls! 
check e.g. https://getbootstrap.com/docs/4.6/components/alerts/#examples (alert-warning)

File: resources/views/frontend/shared-user-collections/show.blade.php [hunk 4]
```diff
@@ -1,51 +1,36 @@
...
-            @endforeach
-        </ul>
+@section('content')
+<div class="container pt-5 mt-5">
+    <div class="mb-5 pb-5 text-lg text-muted">
+        <p>
+            Výber diel z <a href="#TODO" class="underline">Webu <i class="fa fa-search color"></i> umenia</a>.
+        </p>
+        <p class="mt-3">
+            Inšpirované výstavou <a href="#TODO" class="underline">Sedmičky v SNG</a>.
```
Comment (igor-kamil) [inline resources/views/frontend/shared-user-collections/show.blade.php:19]: is it always connected with this exhibition? any condition before?

**Review by igor-kamil — APPROVED** [review]
pls fix this unclosed tag
the rest is fine
I appreciate the tests! 🚀 
I miss the translation strings 😢  (Hope they will come back soon)
lokking forward to see this live!

File: resources/views/components/user-collections/share-form.blade.php (hunk 3 above)
Comment (eronisko) [reply resources/views/components/user-collections/share-form.blade.php:136]: I've tried that but it overstated the importance of the edit link over share link. I'd be worried about people inadvertently sharing their edit links. But I've adjusted the modal a little: 
image

File: resources/views/frontend/shared-user-collections/show.blade.php (hunk 4 above)
Comment (eronisko) [reply resources/views/frontend/shared-user-collections/show.blade.php:19]: cc @ustengrav

**Pull request merged**